js/components/InvestmentList.jsx

```diff
@@ -4,11 +4,31 @@ var InvestmentItem = require('./InvestmentItem');
 var React = require('react');
 
 var InvestmentList = React.createClass({
+  getOrderedInvestments: function() {
+    var comparator = this.getInvestmentComparator();
```

Comment:
Just making sure we're on the same page, `sort()` will also change the value of `this.props.investments`.